plugins/source/slack/resources/plugin/plugin.go

```diff
@@ -0,0 +1,19 @@
+package plugin
+
+import (
+	"github.com/cloudquery/cloudquery/plugins/source/slack/client"
+	"github.com/cloudquery/plugin-sdk/plugins"
+)
+
+var (
+	Version = "development"
```

Comment:
```suggestion
	Version = "Development"
```

Reply:
Sure, but it doesn't matter: https://github.com/cloudquery/plugin-sdk/blob/74c6672e9ebcc6e0e19f309b549be54519be81e1/serve/source.go#L124